linux_sql/psql_docker.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+if [ $# != 2 ];
```

Comment:
the second argument could be optional. if the password is blank, we use the default 'password' as pgpassword.